drivers/media/platform/bcm2835/bcm2835-unicam.c

```diff
@@ -983,11 +983,6 @@ static irqreturn_t unicam_isr(int irq, void *dev)
 		}
 	}
 
-	if (reg_read(unicam, UNICAM_ICTL) & UNICAM_FCM) {
-		/* Switch out of trigger mode if selected */
-		reg_write_field(unicam, UNICAM_ICTL, 1, UNICAM_TFC);
```

Comment:
To someone who knows nothing about the subject matter, the `1` here is unexpected.

Reply:
That's hitting the trigger bit in the ICTL register.  It's using UNICAM_TFC as the bitmask.